fre/pp/checkout_script.py

```diff
@@ -20,6 +20,18 @@
 def checkout_template(experiment = None, platform = None, target = None, branch = None):
     """
     Checkout the workflow template files from the repo
+
+    :param experiment: name of post processing experiment
+    :type experiment: str, default None
+    :param platform: which platform to use
+    :type platform: str, default None
+    :param target: name of target (prod, debug, open-mp, repro)
+    :type target: str, default None
+    :param branch: which git branch to pull from
+    :type branch: str, default None
+    :raises OSError: why checkout script was not able to be created
```

Comment:
It looks like this routine also raises 2 ValueErrors.  I think we should document them here too: you could do something like this: 
```
:raises ValueError: 

    - Description of error 1
    - Description of error 2
```